source/source.go

```diff
@@ -84,7 +84,7 @@ func (s *Source) Parameters() map[string]sdk.Parameter {
 			Description: "The user's password.",
 		},
 		config.KeyAuthDB: {
-			Default:     "admin",
+			Default:     "",
```

Comment:
Because of this, the value of `auth.db` is always set and that makes it impossible to use a MongoDB instance _without_ authentication. This issue also exposed the following two issues/improvements:
1. The connector doesn't use the SDK validation consistently. Some validations (e.g. required) and default values are set using the SDK tags.
2. The acceptance tests do not expose this issue, because the SDK acceptance tests are using the connector sources and destinations directly. When Conduit is interacting with them, it's doing so through an SDK adapter, which is setting these default values.